models/marts/advertising/adwords_ad_performance_adapter.sql

```diff
@@ -13,6 +13,7 @@ joined as (
         criteria_id,
         ad_group_name,
         campaign_id,
+        'google' as utm_source,
         lower(campaign_name) as adwords_campaign_name,
```

Comment:
should this `campaign_name` be the `utm_campaign`?

Reply:
the criteria performance model doesn't have a `utm_campaign` field, just `campaign_name`

Reply:
the `campaign_name` doesn't match necessarily to the `marketing_campaign`

Reply:
gotcha